clap_complete/Cargo.toml

```diff
@@ -40,6 +40,7 @@ is_executable = { version = "1.0.1", optional = true }
 pathdiff = { version = "0.2.1", optional = true }
 shlex = { version = "1.1.0", optional = true }
 unicode-xid = { version = "0.2.2", optional = true }
+clap_builder = { path = "../clap_builder", version = "4.5.13", default-features = false, features = ["unstable-ext"]}
```

Comment:
Why are we depending on `clap_builder`?  `clap` is a facade that we should be working through

Reply:
I want to enable the `unstable-ext` feature for unstable-dynamic, but using `unstable-dynamic=["clap/unstable-ext"]` doesn’t seem to work.

Reply:
I just added it.